### PR TITLE
[12.0][IMP] mail: remove activity_date_deadline column

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
To avoid confusion if someone checks the table later. This field is not stored anymore.

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/mail/migrations/12.0.1.0/end-migration.py [hunk 1]
```diff
@@ -64,7 +64,16 @@ def fill_mail_thread_message_main_attachment_id(env):
         )
 
 
+def remove_activity_date_deadline_column(env):
+    activity_mixin_models = [k for k in env.registry if issubclass(
+        type(env[k]), type(env['mail.activity.mixin'])) and env[k]._auto]
+    _column_renames = {model._table: ('activity_date_deadline', None)
```
Comment (pedrobaeza) [inline addons/mail/migrations/12.0.1.0/end-migration.py]:
```suggestion
    _column_renames = {env[model]._table: ('activity_date_deadline', None)
```

Comment (MiquelRForgeFlow) [reply addons/mail/migrations/12.0.1.0/end-migration.py]: I thought that `model` here is not a string but that `model` is the model class. I was wrong 😑

**Comment by MiquelRForgeFlow** [comment]
updated

**Comment by pedrobaeza** [comment]
Still red

**Pull request merged**